src/include/target/NamimnoRC_FLASH_2400_OLED_TX.h

```diff
@@ -40,9 +41,13 @@
 /* WS2812 led */
 #define GPIO_PIN_LED_WS2812     4
 
-// Output Power
+/* Output Power */
 #define MinPower                PWR_25mW
 #define MaxPower                PWR_1000mW
 #define POWER_OUTPUT_VALUES     {-18,-13,-10,-5,-2,3}
 
+/* Joystick values              {UP, DOWN, LEFT, RIGHT, ENTER, IDLE}*/
+#define JOY_ADC_VALUES          {1850, 900, 490, 1427, 0, 2978}
```

Comment:
@FOG-Yamato are these the correct values. I do not have a NamimnoRC

Reply:
Yes these should be the values necessary.